### DISCUSSION TITLE
blog frontend-website need updates ?

### DISCUSSION THREAD

**Discussion by tjyang** [loco-rs/loco#251] [Q&A] [answered]
@fan-tastic-z , looks like you spent time to go through the blog and create your todolist app.
IMHO, I think [this blog ](https://loco.rs/blog/frontend-website/) need to be updated in a number of sections.
- update 1 by delete, following section already have completed by latest framework.
![image](https://github.com/loco-rs/loco/assets/378638/762f2a7d-d496-4fb3-90ee-42ea48a583ed)

- src/App.rs already have following 4 lines generated correctly. So I think above step is not necessary.
```
   fn routes() -> AppRoutes {
        AppRoutes::with_default_routes()
            .prefix("/api")
            .add_route(controllers::notes::routes())
            .add_route(controllers::auth::routes())
            .add_route(controllers::user::routes())
    }


```
- possible update 2.
production --> development ? 
Not --> Now
![image](https://github.com/loco-rs/loco/assets/378638/eca210ad-3c85-4fcf-bfd4-e0f86c5b6961)

- END

**Comment by fan-tastic-z** [comment]
Indeed, but I understand that this blog is just a very simple example. I am currently trying to write an admin background management system through loco, and I am still learning.

**Comment by kaplanelad** [comment, marked as answer]
@tjyang @fan-tastic-z The blog should not always be with the latest version of the loco. The SaaS starter is bumping automatically when we release a new Loco version.
